lib/src/slider/Slider.accessibility.test.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import { axe } from "jest-axe";
+import DxcSlider from "./Slider.tsx";
+
+// Mocking DOMRect for Radix Primitive Popover
+global.globalThis = global;
+global.DOMRect = {
+  fromRect: () => ({ top: 0, left: 0, bottom: 0, right: 0, width: 0, height: 0 }),
+};
+global.ResizeObserver = class ResizeObserver {
+  observe() {}
+  unobserve() {}
+  disconnect() {}
+};
+
```

Comment:
Is this config needed? I tried to remove from line 6 to 15 and the test passes too.